[2022-06-03 16:03:06,998] {taskinstance.py:826} INFO - Dependencies all met for <TaskInstance: ClusterDAG_test.t1 2022-06-03T16:03:03.479402+00:00 [queued]>
[2022-06-03 16:03:07,044] {taskinstance.py:826} INFO - Dependencies all met for <TaskInstance: ClusterDAG_test.t1 2022-06-03T16:03:03.479402+00:00 [queued]>
[2022-06-03 16:03:07,044] {taskinstance.py:1017} INFO - 
--------------------------------------------------------------------------------
[2022-06-03 16:03:07,044] {taskinstance.py:1018} INFO - Starting attempt 1 of 1
[2022-06-03 16:03:07,044] {taskinstance.py:1019} INFO - 
--------------------------------------------------------------------------------
[2022-06-03 16:03:07,073] {taskinstance.py:1038} INFO - Executing <Task(BashOperator): t1> on 2022-06-03T16:03:03.479402+00:00
[2022-06-03 16:03:07,078] {standard_task_runner.py:51} INFO - Started process 5149 to run task
[2022-06-03 16:03:07,083] {standard_task_runner.py:75} INFO - Running: ['airflow', 'tasks', 'run', 'ClusterDAG_test', 't1', '2022-06-03T16:03:03.479402+00:00', '--job-id', '145', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/HelloDAG.py', '--cfg-path', '/tmp/tmplffnjw1k']
[2022-06-03 16:03:07,083] {standard_task_runner.py:76} INFO - Job 145: Subtask t1
[2022-06-03 16:03:07,248] {logging_mixin.py:103} INFO - Running <TaskInstance: ClusterDAG_test.t1 2022-06-03T16:03:03.479402+00:00 [running]> on host slave01
[2022-06-03 16:03:07,396] {taskinstance.py:1232} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=ClusterDAG_test
AIRFLOW_CTX_TASK_ID=t1
AIRFLOW_CTX_EXECUTION_DATE=2022-06-03T16:03:03.479402+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-06-03T16:03:03.479402+00:00
[2022-06-03 16:03:07,396] {bash.py:135} INFO - Tmp dir root location: 
 /tmp
[2022-06-03 16:03:07,397] {bash.py:158} INFO - Running command: sleep 1
[2022-06-03 16:03:07,406] {bash.py:169} INFO - Output:
[2022-06-03 16:03:07,442] {bash.py:173} INFO - /root/.bashrc: line 18: zoo-start: command not found
[2022-06-03 16:03:08,443] {bash.py:177} INFO - Command exited with return code 0
[2022-06-03 16:03:08,569] {taskinstance.py:1142} INFO - Marking task as SUCCESS. dag_id=ClusterDAG_test, task_id=t1, execution_date=20220603T160303, start_date=20220603T160306, end_date=20220603T160308
[2022-06-03 16:03:08,664] {taskinstance.py:1195} INFO - 2 downstream tasks scheduled from follow-on schedule check
[2022-06-03 16:03:08,698] {local_task_job.py:118} INFO - Task exited with return code 0
